static/redfish/v1/schema/OemLogEntry_v1.xml

```diff
@@ -33,6 +33,10 @@
               <Annotation Term="OData.LongDescription" String="A true value says external interface acked error log, false says otherwise."/>
             </Property>
 
+            <Property Name="InitiatedBy" Type="Edm.String">
```

Comment:
Shouldn't this not be a string then and should be a resource of type URI. Please look at how Redfish does other links

Reply:
Client can use this id and build the session url if they want to. 
-OR- for extendibility any other LogService resource can use some other user identifier here - which is not be a url.

Reply:
Done

Reply:
Redfish is most likely going to do a link. They try to avoid free strings like this because they aren't interoperable. I really don't think a string makes sense here

Reply:
If its a link, that will limit the usage of the GeneratorId to assign only the GETtable redfish endpoints. If a session logs out after triggering a dump, the GET on that link will fail. 
Due to this point, it was decided to use the IP address of the client as a "GeneratorId"